packages/components/src/components/breadcrumb/test/html.mock.ts

```diff
@@ -35,7 +35,7 @@ export const getBreadcrumbHtml = (props: KoliBriBreadcrumbProps): string => {
 												_ariaLabel: link._ariaLabel as string,
 												_icon: link._icon as Icofont,
 										  })
-										: link._label
+										: link._label || link._href
```

Comment:
keine änderungen in diese Komponente nötig

Reply:
Wenn das Label wieder Pflicht ist, ist diese Änderung nicht mehr nötig.

Reply:
Aber _label ist nicht mehr Pflicht?